pyomo/solvers/plugins/converter/ampl.py

```diff
@@ -78,7 +78,8 @@ def apply(self, *args, **kwargs):
         #
         # Execute command and cleanup
         #
-        output = pyutilib.subprocess.run(cmd)
+        output = subprocess.run(cmd, stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
         if not os.path.exists(output_filename):       #pragma:nocover
-            raise ApplicationError("Problem launching 'ampl' to create '%s': %s" % (output_filename, output))
+            raise ApplicationError("Problem launching 'ampl' to create '%s': %s" % (output_filename, output.stdout.decode("utf-8")))
```

Comment:
This only outputs stdout.  It might be better to make the run call use `stderr=subprocess.STDOUT` so that we get both stdout and stderr.